docs/mounting.md

```diff
@@ -1,179 +1,202 @@
-# Credentials
+# Prerequisites
 
-Credentials for use with GCS will automatically be loaded using [Google
-application default credentials][app-default-credentials], unless the flag
-`--key-file` is set to a path to a JSON key file downloaded from the Google
-Developers Console.
+1. Before invoking Cloud Storage FUSE, you must have a Cloud Storage bucket that you want to mount. If you haven't yet, [create](https://cloud.google.com/storage/docs/creating-buckets#storage-create-bucket-console) a storage bucket. 
+2. Provide [Application Default Credentials](https://cloud.google.com/docs/authentication/application-default-credentials#howtheywork) to authenticate Cloud Storage FUSE requests to Cloud Storage. By default, Cloud Storage FUSE automatically loads the Application Default Credentials without any further configuration if they exist. You can use the gcloud auth login command to easily generate Application Default Credentials.
 
-The easiest way to set up credentials when running on [Google Compute
-Engine][gce] is to create your VM with a service account using the
-`storage-full` access scope. (See [here][gce-service-accounts] for details on
-VM service accounts.) When gcsfuse is run from such a VM, it automatically has
-access to buckets owned by the same project as the VM.
+        gcloud auth application-default login
+        gcloud auth login
+        gcloud auth list
 
-When testing, especially on a developer machine, credentials can also be
-configured using the [gcloud tool][]:
+   Alternatively, you can authenticate Cloud Storage FUSE by setting the --key-file flag to the path of a JSON key file, which you can download from the Google Cloud console. You can also set the GOOGLE_APPLICATION_CREDENTIALS environment variable to the path of the JSON key.
```

Comment:
nit: Even though not in user guide, you can put --key-file and GOOGLE_APPLICATION_CREDENTIALS in code block

Reply:
fixed